poc.md

```diff
@@ -0,0 +1,155 @@
+# Proof of Concept for Layer 2 Networking in Orchestrator
+
+This file contains notes and code for proving the networking design as defined in the [EMT-S Scale Provisioning design
+proposal](https://github.com/open-edge-platform/edge-manageability-framework/blob/4d801d24d402871ee664fbedcd3f964d43a784b2/design-proposals/emts-scale-provisioning.md?plain).
+
+## Deployment
+
+### Prerequisites
+
+1. On-premise Orchestratror fully deployed on the system
+   1. Edge Network is configured with a bridge interface `virbr1`
+1. `kubectl` is configured with root permissions to manage the Kubernetes cluster
+1. `nmap` is installed on the system to verify network connectivity
+   1. On Ubuntu, install with `sudo apt install nmap`
+
+### Steps
+
+1. Create a pod the deploys a DHCP server with host networking
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dhcp-server
+  namespace: default
+spec:
+  hostNetwork: true  # Use the host's network namespace
+  containers:
+  - name: dhcp-server
+    image: alpine:latest
+    # env:
+    # - name: http_proxy
+    #   value: ""  # Optional, if you need to set a proxy
+    # - name: https_proxy
+    #   value: ""  # Optional, if you need to set a proxy
+    # - name: no_proxy
+    #   value: "localhost,127.0.0.1"
+    ports:
+      - containerPort: 67
+        hostPort: 67
+        protocol: UDP
+      - containerPort: 68
+        hostPort: 68
+        protocol: UDP
+    command: ["sh", "-c"]
+    args:
+    - |
+      # Install dnsmasq for DHCP server and PXE boot options
+      apk add --no-cache dnsmasq
+
+      # Create dnsmasq configuration file to add PXE TFTP server option
+      cat > /etc/dnsmasq.conf << EOF
+      # Enable logging for troubleshooting
+      log-queries
+      log-dhcp
+
+      # Disable DNS functionality, we only want DHCP
+      port=0
+
+      # Enable DHCP server functionality - assign addresses directly
+      dhcp-range=192.168.99.100,192.168.99.200,255.255.255.0,12h
+
+      # Use the node's network interface that is connected to network where Edge Nodes will connect
+      interface=ens3 # Replace with your actual network interface name
+      bind-interfaces
+
+      # PXE boot options configuration
+      # Option 66 (next-server) and Option 67 (bootfile-name)
+      dhcp-option-force=66,192.168.99.40
+      dhcp-option-force=67,signed_ipxe.efi
+
+      # Alternative PXE boot configuration - works with most PXE clients
+      dhcp-boot=signed_ipxe.efi,tinkerbell-nginx.cluster.onprem,192.168.99.40
```

Comment:
Has this been verified? It's an HTTPS endpoint, which is not supported by PXE firmware